### PR TITLE
add runs download doc

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/docs/cli-command-reference/runs-download.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+For example, if the `runs download` command is run on the 6th of May 2023 at 10.30 am, against test _U5432_ which is still running, the created folder name would look similar to the following example:
+
+```
+U5432-2023-05-06_10:30:15
+```
+
+A message is displayed to show the location of any downloaded artifacts from the incomplete run, for example:
+
+```
+GAL1xxxxI: Downloaded 11 files to folder U5432_22023-05-06_10:30:15
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-download.md:60]: The GAL1xxxxI should be GAL2501I

Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-download.md:60]: Also, the 22023 should be 2023

File: src/markdown-pages/docs/cli-command-reference/runs-download.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+### Identifying output for test retries
+
+If a test is scheduled to run at a particular time but is unable to start, for example due to a lack of resources that are available in the environment, the test can be retried at a later time. When investigating problems with a test running in an ecosystem, it is useful to download the artifacts that are associated with the retries of that test.
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/runs-download.md:65]: > the test can be retried at a later time

The ecosystem may retry the test at a later time.

File: src/markdown-pages/docs/cli-command-reference/runs-download.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-download.md:60]: updated

Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/runs-download.md:60]: oops, thank you

**Review by AaronJhaj — APPROVED** [review]

**Pull request merged**